src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/Args.java

```diff
@@ -52,14 +52,13 @@
   }
 
   /**
-   * @return [--name, value1, --name, value2, ...] or [] if value=null.
+   * @return [--name=value1, --name=value2, ...] or [] if value=null.
    */
-  static List<String> strings(String name, List<String> values) {
+  static List<String> stringsEq(String name, List<String> values) {
```

Comment:
please use  amore complete method name. I'd guess this is testing strings for equality but it doesn't seem to be.